package Desafio6.Exceptions.ExceptionsKind;

public class UserNotFoundException extends RuntimeException{
    public UserNotFoundException(String message){super(message);}
}
